_scripts/stop-containers.sh

```diff
@@ -0,0 +1 @@
+docker compose down
```

Comment:
maybe it is a good idea to kill the leftover volumes and prune all stuff after you down them.